packages/test-generated/spec/reflection-type-check.spec.ts

```diff
@@ -0,0 +1,235 @@
+import {
+    assert,
+    isOneofGroup,
+    normalizeFieldInfo,
+    IMessageType,
+    PartialMessage,
+    ReflectionTypeCheck,
+    ScalarType,
+    UnknownMessage
+} from "@protobuf-ts/runtime";
+import {
+    RepeatedScalarValuesMessage,
+    ScalarValuesMessage
+} from "../ts-out/msg-scalar";
+import {EnumFieldMessage} from "../ts-out/msg-enum";
+import {MessageMapMessage, ScalarMapsMessage} from "../ts-out/msg-maps";
+import {OneofMessageMemberMessage, OneofScalarMemberMessage} from "../ts-out/msg-oneofs";
+import {TestAllTypesProto3} from "../ts-out/google/protobuf/test_messages_proto3";
+import {TestAllTypesProto2} from "../ts-out/google/protobuf/test_messages_proto2";
+
+
+describe('ReflectionTypeCheck.is()', function () {
+    const types: IMessageType<any>[] = [
+        EnumFieldMessage,
+        ScalarMapsMessage,
+        MessageMapMessage,
+        OneofScalarMemberMessage,
+        OneofMessageMemberMessage,
+        TestAllTypesProto3,
+        TestAllTypesProto2,
+    ];
+
```

Comment:
This looks unused?

Reply:
Grr, sorry. Moving too fast. Guess I rely too much on eslint.